packages/codium.rb

```diff
@@ -3,16 +3,17 @@
 class Codium < Package
   description 'VSCodium is Open Source Software Binaries of VSCode with a community-driven default configuration.'
   homepage 'https://vscodium.com/'
-  version '1.49.2'
   compatibility 'aarch64,armv7l,x86_64'
   case ARCH
   when 'aarch64', 'armv7l'
+    version '1.49.1'
```

Comment:
`version '1.49.3'`?

Reply:
arm hasn't been updated, only amd64 has
> https://github.com/skycocker/chromebrew/pull/4362#issuecomment-703130291

Reply:
Oh, I see from the earlier thread.  This is why modern coders don't have a clue.  Don't release until it works for all architectures.  I would like to hold off on this and give them time to repent from straying off the beaten path.